test/test_images/grpc-ping

```diff
@@ -0,0 +1 @@
+/Developer/work/go/src/knative.dev/client/vendor/knative.dev/serving/test/test_images/grpc-ping
```

Comment:
It should be relative link, something like the following should do the trick.
```
ln -s -r vendor/knative.dev/serving/test/test_images/grpc-ping test/test_images/grpc-ping
```

Reply:
i dont see a -r option in mac.. not sure what am missing

Reply:
I guess older `ln` version vs. gnu `ln`. For me It works also relatively to link without `-r` option.

```
cd test/test_images
ln -s ../../vendor/knative.dev/serving/test/test_images/grpc-ping grpc-ping
```